### PR TITLE
{chem} [intel/2015a] PSI4 git rev 3815c4672b2 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
Superseeds #1367.
Needs  hpcugent/easybuild-easyblocks#545

### CODE REVIEW

**Comment by wpoely86** [comment]
please retest

**Comment by wpoely86** [comment]
retest this please

File: easybuild/easyconfigs/p/PSI/PSI-4.0b6-3815c4672b2-intel-2015a-mt.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name = 'PSI'
+commit = '3815c4672b2'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PSI/PSI-4.0b6-3815c4672b2-intel-2015a-mt.eb]: use a datestamp as a versionsuffix rather a meaningless/ugly commit ID in the module name?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ebd1765d5ae4a7fc7b4d for a full test report.

**Comment by wpoely86** [comment]
@boegel 
```Module 'ncurses/5.9-GCC-4.9.2' conflicts with the currently loaded module(s) 'ncurses/5.9-intel-2015a'```.
I've done all this with Lmod...

**Comment by boegel** [comment]
For now, you'll need to stick to `intel/2015a` for CMake...

I do wonder why the unit tests didn't pick this up, they should.

**Comment by wpoely86** [comment]
@boegel fixed

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/493f89ec671e34e708df for a full test report.

**Comment by boegel** [comment]
@wpoely86:

```
CMake Error at doc/sphinxman/CMakeLists.txt:308 (add_custom_command):
  The target name "sphinxpdf" is unknown in this context.
```

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/acb77ee12fe338c35f5c for a full test report.

**Comment by wpoely86** [comment]
@boegel that's an error in their `CMakeLists.txt`. Should be fixed now.

**Comment by boegel** [comment]
@wpoely86: can you look into a test report please?

**Comment by wpoely86** [comment]
@boegel it's running as we speak :wink:

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/f46d2e742a1d4ba556c0 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/aaf30118dcfa0a15c687 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.9
See https://gist.github.com/6fc0edb9968345e1135e for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/77c11011cfda8ea874f5 for a full test report.

**Comment by wpoely86** [comment]
@boegel ready, tests work!

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/3b9b543452ab095b2a49 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4b96a4b96c89e84d5a49 for a full test report.

**Comment by boegel** [comment]
https://github.com/hpcugent/easybuild-easyblocks/pull/545 is merged, this works fine, so going in as well. Thanks @wpoely86!

**Pull request merged**